sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTraceExporter.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Exporter.AzureMonitor
+{
+    public class AzureMonitorTraceExporter : ActivityExporter
```

Comment:
can be internal. Every other exporter is internal in Otel main repo. Users get it through extension methods on Builder.